client/blocks/design-sidebar/index.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import RootChild from 'components/root-child';
+
+class DesignSidebar extends Component {
+	constructor( props ) {
+		super( props );
+	}
+
+	render() {
+		const className = 'design-sidebar' + ( this.props.isVisible ? ' is-visible' : '' );
+		return (
+			<RootChild>
+				<div className={ className }>
+				</div>
+			</RootChild>
+		);
+	}
+}
+
+DesignSidebar.propTypes = {
+	isVisible: PropTypes.bool,
+};
+
+DesignSidebar.defaultProps = {
+	isVisible: false,
+};
+
+export default DesignSidebar;
+
```

Comment:
It's a wrapper for `DesignMenu` which has yet to be yet to be added (#4429). It's not necessary to add it in this PR; that could be done later.